namer/namer.cc

```diff
@@ -629,12 +629,17 @@ class SymbolFinder {
         return foundDefs->addTypeMember(move(found));
     }
 
+    bool isT(const ast::ExpressionPtr &expr) {
```

Comment:
We just put this here initially, directly above the `handleAssignment` method that uses it.

Where should a helper like this belong?

Reply:
Here is fine. We prefer not to over-abstract helpers like this.

Could you make this also check for resolved constants? Most constants are not yet resolved by namer, but sometimes resolved `T` constants are created by rewriter phases.

you'd want to add a case for `ast::ConstantLit` and check that the symbol inside it is equal to `core::Symbols::T()`.